Makefile

```diff
@@ -572,10 +572,13 @@ vendor-in-container:
 package:  ## Build rpm packages
 	## TODO(ssbarnea): make version number predictable, it should not change
 	## on each execution, producing duplicates.
-	rm -f  ~/rpmbuild/RPMS/x86_64/* ~/rpmbuild/RPMS/noarch/*
+	rm -rf build/* *.src.rpm ~/rpmbuild/RPMS/*/*
 	./contrib/build_rpm.sh
 
+# Remember that rpms install exec to /usr/bin/podman while a `make install`
+# installs them to /usr/local/bin/podman which is likely before. Always use
+# a full path to test installed podman or you risk to call another executable.
 package-install: package  ## Install rpm packages
-	sudo ${PKG_MANAGER} -y remove podman podman-remote
-	sudo ${PKG_MANAGER} -y clean all
-	sudo ${PKG_MANAGER} -y install ${HOME}/rpmbuild/RPMS/*/*.rpm
+	sudo ${PKG_MANAGER} -y install --allowerasing ${HOME}/rpmbuild/RPMS/*/*.rpm
```

Comment:
I don't think we should use `--allowerasing`. From what I could test, it is not an option that `yum` will understand, only `dnf`.

Reply:
@NevilleC I added it because at some point I encountered a case where yum picked podman-1.0 from system instead of the newly build rpm, and this is not something we want to happen.

So far none of the 3 platform jobs we had complained about that option so I do not find it as a bug. Do you know a specific platform-version what needs to be supported so I can enable an extra job for it?